Build/Linux/DEBIAN/control

```diff
@@ -0,0 +1,14 @@
+Package: AtomicGameEngine
+Version: 0.0.8
+Installed-Size: 393064
+Section: graphics
```

Comment:
Its a toss up between Programming or Graphics. I picked Graphics because that's where blender and other 3d modelling and art programs ended up. Its a one word change, if you want it moved.